src/installer/managed/Microsoft.NET.HostModel/Bundle/FileSpec.cs

```diff
@@ -20,8 +20,8 @@ public class FileSpec
 
         public FileSpec(string sourcePath, string bundleRelativePath)
         {
-            SourcePath = sourcePath;
-            BundleRelativePath = bundleRelativePath;
+            SourcePath = NormalizeDirectorySeparator(sourcePath);
```

Comment:
Can this be Windows UNC path? If yes, is the normalization going to work well for Windows UNC paths?

Reply:
Yes. I think this could break paths like `\\x\...`. We already normalize relative paths in `FileEntry`, I made the change there, more economic/safe.